syntax = "proto3";

package com.dipper.proto.rpc;

option java_package = "com.dipper.proto.rpc";
option java_outer_classname = "BlogProto";
option java_multiple_files = true;

message BlogPro {
    int32 id = 1;
    int32 user_id = 2;
    string header = 3;
    string content = 4;
    string image = 5;
    string build_time = 6;
 }